File: Pizza.cpp
#include "Pizza.h"

Pizza::Pizza(int new_id, std::string new_name, int new_price, int new_eatingTime, int new_prepareTime) : 
Product(new_id, new_name, new_price, new_prepareTime)
{
    eatingTime = new_eatingTime;
}

int Pizza::getEatingTime()
{
    return eatingTime;
}

File: simulation_prototype/Pizza.py
from Product import Product


class Pizza(Product):
    def __init__(self, new_id, new_name, new_price, new_eatingTime, new_prepareTime):
        super().__init__(new_id, new_name, new_price, new_prepareTime)

        self._eatingTime = new_eatingTime


    def __str__(self):
        return "NAME:" + self._name


    def getEatingTime(self):
        return self._eatingTime
